README.md

```diff
@@ -16,7 +26,47 @@ You can also opt to use npx:
 npx -p @govtechsg/open-attestation-cli open-attestation <arguments>
 ```
 
-## Wrapping Documents
+> In all the guides, we will refer to the CLI as `open-attestation` when running a command. That means we will assume the CLI is available in your execution path. If it's not the case, you will to change `open-attestation` by the full path to the executable.
+
+---
+
+## Usage
+
+#### List of features with the options available
+
+|                                            | Private Key | Wallet | Aws Kms |
+| ------------------------------------------ | ----------- | ------ | ------- |
```

Comment:
the option columns are args that must be provided in order to use the feature yeah?

```suggestion
#### List of features with the required options

| Feature                                    | Private Key | Wallet | Aws Kms |
| ------------------------------------------ | ----------- | ------ | ------- |
```

Reply:
yup. ok have changed it.